src/style.css

```diff
@@ -1,3 +1,89 @@
+html {
+  margin: 0;
+  display: block;
+
+  --main-color: #00acdc;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). The `html` selector here is unnecessary and can be removed since it only contains a variable declaration which can be declared globally without targeting the `html` tag.